internal/cmd/base/servers.go

```diff
@@ -839,6 +839,11 @@ func (b *Server) SetupWorkerPublicAddress(conf *config.Config, flagValue string)
 	if err != nil {
 		return fmt.Errorf("Error splitting public adddress host/port: %w", err)
 	}
+	if host != "" {
+		if host, err = parseutil.NormalizeAddr(host); err != nil {
+			return fmt.Errorf("Error normalizing worker address")
+		}
+	}
```

Comment:
Oops seems like we have some [overlapping](https://github.com/hashicorp/boundary/pull/5587/commits/7da73ee596a61196c1398b346d743131afeac196) work since I was looking at the `base` package too

Reply:
yeah I think this is what we were talking about the other day regarding worker startup. I think we ended up coming to the same conclusion, though, so we can just handle it as a merge conflict later on